index.html

```diff
@@ -26,12 +27,12 @@
             <div id="photogrammatron" class="row">
                 <div id="left-panel" class="col-xs-3 well">
                     <h3>Select File</h3>
-                    <form>
+                    <form id="add">
```

Comment:
this id should be more descriptive - short, generic ids make it easy to duplicate them, which breaks the performance gained by having unique keys in a hash table (dictionary)